src/Adapter/Order/Refund/OrderSlipCreator.php

```diff
@@ -252,7 +252,7 @@ private function createOrderSlip(
             }
 
             $product_tax_incl_line = Tools::ps_round($tax_calculator->{$add_or_remove . 'Taxes'}($price) * $quantity, $precision);
-
+            $product_tax_incl = 0;
```

Comment:
Same, declare this variable in the default satement

Reply:
:heavy_check_mark: